en/admin/config.rst

```diff
@@ -1750,6 +1754,11 @@ Thread management can be configured by threads parameters. The type and value ra
 
     If **thread_worker_pooling** parameter is true, all threads used for client requests execution are pooled on server boot.
 
+**thread_core_count**
+
+    The number of groups of pooled threads is configured according to the **thread_core_count** parameter. The default value is set to the number of system cores.
+    If the number of threads in a group does not reach 3 or more according to the parameter value, the system adjusts this value so that at least 3 thread belong to each group.
```

Comment:
```suggestion
    If the number of threads in a group does not reach 3 or more according to the parameter value, the system adjusts this value so that at least 3 threads belong to each group.
```

Reply:
Thank you so much!